### ISSUE TITLE
Add DeArrow as an opt-in feature

### ISSUE THREAD

**Issue by GeorgeWL** [polymorphicshade/Tubular#170]
### Checklist

- [X] I made sure that there are *no existing issues* - [open](https://github.com/polymorphicshade/Tubular/issues) or [closed](https://github.com/polymorphicshade/Tubular/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I have read the [FAQ](https://newpipe.net/FAQ/) and my problem isn't listed.
- [X] I'm aware that this is a request for Tubular itself and that requests for adding a new service need to be made in [vanilla NewPipeExtractor](https://github.com/TeamNewPipe/NewPipeExtractor/issues).
- [X] I have taken the time to fill in all the required details. I understand that the feature request will be dismissed otherwise.
- [X] This issue contains only one feature request.
- [X] I have read and understood the vanilla NewPipe [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/dev/.github/CONTRIBUTING.md).

### Feature description

I think it's a generally quite useful plugin for YouTube, and should in theory (though I've yet to test it) work with Tubular too

The plugin takes video titles that are overly clickbait and rewrites them with a simple descriptive title (from crowdsourced) 

E.g. "You wouldn't BELIEVE this Pokémon B&W speedrun!!!! No hacks! Legit?‽!" to "Alpharad explains a lesser-known speedrun tactic for Pokémon Black & White"

### Why do you want this feature?

I find it can be frustrating to have video titles that don't really say what's in the video

### Why ist the feature relevant to this fork?

Features like Sponsorblock and DeArrow are against the core ethos of NewPipe main branch that it should simply be a ad-free wrapper of YouTube and nothing more

**Comment by asandikci** [comment]
Agreed, I think already on todo list https://github.com/polymorphicshade/Tubular/blob/master/README.md#L11

**Comment by asandikci** [comment]
+ Duplicate of #4
